### ISSUE TITLE
(BUG) Synth inability to repair self

### ISSUE THREAD

**Issue by Kegsey** [VOREStation/VOREStation#7482]
#### Brief description of the bug
Inability to fix self as a synth, I assume it's to do with Aronai's inducer fix?

#### What you expected to happen
Click self with welder with <15 damage
be fixed

#### What actually happened
Nothing

#### Steps to reproduce
- (Step 1) Be damaged
- (Step 2) Try to fix treatable damage
- (Step 3) Have it not work

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)

 ff769d941c244c368fb81225bc02952f7d032676 
#### Anything else you may wish to add:
- (Location if it's a mapping issue, screenshots, sprites, etc.)

**Comment by Arokha** [comment]
Wat, how would this be related to inducer change? That was only for the inducer.

**Comment by Arokha** [comment]
![image](https://user-images.githubusercontent.com/15028025/80400778-4c497100-8889-11ea-92e2-7a02b437406c.png)

**Comment by KasparoVy** [comment]
Prakash coming in clutch

**Comment by Heroman3003** [comment]
Were you in the voidsuit at the moment? Synths cant do repair through them.

**Comment by Kegsey** [comment]
I think I can close this issue actually, as later in the round I recalled that my chest had never been closed; I assumed that due to the intent addition it could've been that Aronai's inducer updoot but I'm dumb 'n' didn't read the code for it.

TL;DR: I'm dumb
